supervisor/addons/validate.py

```diff
@@ -336,7 +338,17 @@ def _migrate(config: dict[str, Any]):
         vol.Optional(ATTR_DEVICES): [str],
         vol.Optional(ATTR_UDEV, default=False): vol.Boolean(),
         vol.Optional(ATTR_TMPFS, default=False): vol.Boolean(),
-        vol.Optional(ATTR_MAP, default=list): [vol.Match(RE_VOLUME)],
+        vol.Optional(ATTR_MAP, default=list): [
+            vol.Any(
+                vol.Match(RE_VOLUME),
+                vol.Schema(
+                    {
+                        vol.Required('name'): str,
+                        vol.Optional('target'): str
+                    }
+                )
+            )
+        ],
```

Comment:
Only allow one format (dict) and convert the other over to this format

Reply:
Would the existing format (config:rw) be considered deprecated?